packages/babel-plugin-transform-async-generator-functions/test/fixtures/for-await/async-generator-exec/exec.js

```diff
@@ -22,5 +24,7 @@ var output = 0;
 return forEach(genAnswers(), function(val) { output += val.value })
 .then(function () {
   assert.equal(output, 42);
+})
+.catch(function (error) {
```

Comment:
If we wanted to test .catch we need to throw an error in a separate test so that we can test both a .then and a .catch separately? I thought that it shouldn't work because I didn't implement that as mentioned in the issue